solutions/banking/watson-scripts/assistant-read.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+set -e
+
+project_name="gen-ai-rag-sample-app-assistant"
```

Comment:
This should be an env var or come from the state data (see previous comment)
Probably it's more robust to fetch resource ID from the input and use that in the API call.
Also, fetch other things like integration ID that are needed and merge them into the json data, filter out any attributes that might cause a diff between terraform runs (e.g. timestamps, etc)